src/components/Form/FormField.jsx

```diff
@@ -16,8 +16,8 @@ export default function FormField({
   const nonEditingStyles =
     "outline-none p-0 border-none bg-transparent shadow-none appearance-none";
   const editingStyles = `w-full p-1 pt-[6px] pl-3 ${
-    errors && errors[keyLabel] && "border-red-500 border-2"
-  } focus:outline-none`;
+    errors && errors[keyLabel] && "!border-error-red"
+  } focus:outline-0 focus:ring-0 focus:border-ca-pink`;
```

Comment:
I would leave the focus color as default because ca-pink looks pretty close to the error color we use, so it could confuse users

Reply:
Makes sense, we could make it a darker gray instead or disable the textbox focus as well, the default blue looks kind of out-of-place but it might look different on Windows than Mac

Reply:
I might be understanding this wrong but in the design system the focus color is black for an active input field so it would be best to stick with that one
<img width="689" alt="Screenshot 2024-02-20 at 8 24 59 AM" src="https://github.com/GTBitsOfGood/canine-assistants/assets/52293714/407e7083-db53-4f1f-9859-0ffd4e1803ca">